recipes/samwell/meta.yaml

```diff
@@ -31,9 +31,8 @@ requirements:
     - typing_inspect >=0.3.1
     - attrs >=19.3.0
     - defopt >=6.0
-    - intervaltree >=3.0.2
     - mypy_extensions >=0.4.3
-    - pybedtools >=0.8.1
+    - pybedlite >=0.0.1
```

Comment:
Looks like this will need to be added.

Reply:
Huh, looks like it's still failing. Maybe related to the fact that for some reason pybedlite conda seems to think the minimum python version is 3.10? Which is odd, because in the recipe for pybedlite it calls for a minimum of 3.6
https://bioconda.github.io/recipes/pybedlite/README.html

https://github.com/bioconda/bioconda-recipes/pull/32538/files

Reply:
Oh, pybedlite only got built for OSX for some reason. I'll check into that.